### PR TITLE
Fix tyop

### PR DESCRIPTION
**Pull request by da4089**

### CODE REVIEW

File: docs/how.rst [hunk 1]
```diff
@@ -16,7 +16,7 @@ emerging software-inspired paradigms like
 `Migen ➚ <https://m-labs.hk/gateware/migen/>`_, or
 :gh:`Amaranth ➚ <amaranth-lang>`.
 Since early 2000s, free and open source tools allow simulating HDLs.
-However, for several decades the major problem lied in the frontend and backend, where there was no established
+However, for several decades the major problem lay in the frontend and backend, where there was no established
```
[the remark this answers is not in the document]
Comment (da4089) [reply docs/how.rst:19]: I'm not a grammar expert, so take this with some skepticism, but:

First off, _lie_ and _lay_ are different.  To _lay_ something is to actively place it, usually horizontally.  To _lie_ is more the ongoing act of being horizontal (also. not telling the truth, but that's an entirely different word).

In the case above, the problem is engaged in an ongoing act.  So in the present, you could say "the problem _lies_ in the frontend and backend".  Confusingly, I think the past tense of _lie_ is _lay_, and so my suggested change.

"was" might be a simpler replacement.

[the remark this answers is not in the document]
Comment (da4089) [reply docs/how.rst:19]: Ok.  I found this.  I think it agrees with my comment above: https://www.merriam-webster.com/words-at-play/how-to-use-lay-and-lie

[the remark this answers is not in the document]
Comment (umarcor) [reply docs/how.rst:19]: @da4089 english is not my mother tongue, so I appreciate your time to clarify. I learnt something today!

**Review by umarcor — APPROVED** [review]

File: docs/how.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (da4089) [reply docs/how.rst:19]: English is my first (and by far best) language, and it wasn't an easy topic for me either!  Thanks for your effort on what is ultimately only an extremely small thing in a much larger (and very impressive) project.

**Pull request merged**